docs/09-implementation-roadmap.md

```
@@ -0,0 +1,1037 @@
+- Popular publication track (simplified claims, public misconceptions)
+- Full career role progression (through professor)
+- All career activity types (conferences, grants, sabbatical)
+- Richer NPC personalities and relationships
+- The desk-based UI evolution (Strange Horticulture aesthetic)
+- Cultural profile document type (player-authored culture models)
+
+-----
+
+## Dependency Graph
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)